register.php

```diff
@@ -27,16 +27,16 @@
 			<div class="col text-center">
 				<img src="logo.png" class="my-4" alt="Stravan't">
 				<h1>Register a new account</h1>
-				<form method="POST">
+				<form method="POST" action="create-account.php">
 					<?php if ($error): ?>
 						<div class="alert alert-danger" role="alert">
 							<?=$error?>. Maybe it would be easier to just <a href="steal.php">steal an account</a>?
 						</div>
 					<?php endif; ?>
 					<div class="form-group row">
-						<label for="username" class="col-sm-2 col-form-label">Username</label>
+						<label for="email" class="col-sm-2 col-form-label">Email</label>
 						<div class="col-sm-10">
-							<input type="text" class="form-control" id="username" name="username">
+							<input type="text" class="form-control" id="email" name="email">
```

Comment:
There's actually an input type="email" which should do some validation work for us